INN/umbrella-ipbs
# Homepage header redo: now with Widgets™!

## Changes

This pull request makes the following changes:

- replaces the former featured image on the homepage with one that is static, served from images/ with the choice of a 500KB or a 100KB image.
- replace the former featured post on the top of the homepage with a Homepage Top widget area, designed for a Text Widget with title, linked title, and some text.

<img width="1035" alt="Screen Shot 2020-02-03 at 18 59 48" src="https://user-images.githubusercontent.com/1754187/73701652-3d9f5380-46b8-11ea-936d-2a55bacf3c95.png">
<img width="391" alt="Screen Shot 2020-02-03 at 19 00 51" src="https://user-images.githubusercontent.com/1754187/73701653-3d9f5380-46b8-11ea-9fd0-f4213f1b3959.png">


## Why



For #55, where we'd had a post before and missed the fact that it should've been written for something other than a post.

## Testing/Questions

Features that this PR affects:

- Homepage
- Widgets


Questions that need to be answered before merging:

- [ ] Do we have a better copy of this image? I pulled this one from Adobe XD by inspecting network traffic. It's a moderately massy image, at half a megabyte.
- [x] Are we worried about them trying to put multiple widgets in this area?
- [ ] Are there specific markup cases like `.btn` that need to be tested here?
- [x] Should the Quick Links menu in this area be part of the widget area?

Steps to test this PR:

1. Check out the branch
2. Add a text widget to the Homepage Top widget area, with a title, some text, and a link affecting the title.
3. View the homepage at a variety of screen sizes.
